content/bn/community/static/cncf-code-of-conduct.md

```diff
@@ -0,0 +1,85 @@
+<!-- Do not edit this file directly. Get the latest from
+     https://github.com/cncf/foundation/blob/main/code-of-conduct.md -->
```

Comment:
That's not the right link. Did you mean https://github.com/cncf/foundation/blob/main/code-of-conduct-languages/bn.md?

Reply:
Yes.